prow/plugins/docs-no-retest/docs-no-retest.go

```diff
@@ -35,9 +35,10 @@ const (
 )
 
 var (
-	docFilesRegex     = regexp.MustCompile(`^.*\.(md|png|svg|dia)$`)
-	ownersFilesRegex  = regexp.MustCompile(`^OWNERS$`)
-	licenseFilesRegex = regexp.MustCompile(`^LICENSE$`)
+	docFilesRegex         = regexp.MustCompile(`^.*\.(md|png|svg|dia)$`)
+	ownersFilesRegex      = regexp.MustCompile(`^OWNERS$`)
+	licenseFilesRegex     = regexp.MustCompile(`^LICENSE$`)
+	securityContactsRegex = regexp.MustCompile(`^SECURITY_CONTACTS$`)
```

Comment:
Would it be better to drive these via config?

Reply:
Probably, yes.